autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py

```diff
@@ -51,7 +51,7 @@ class MultimodalWebSurferAgent(ConversableAgent):
 
     DEFAULT_DESCRIPTION = "A helpful assistant with access to a web browser. Ask them to perform web searches, open pages, and interact with content (e.g., clicking links, scrolling the viewport, etc., filling in form fields, etc.)"
 
-    DEFAULT_START_PAGE = "https://www.bing.com/"
+    DEFAULT_START_PAGE = "https://www.berkshirehathaway.com/"
```

Comment:
This seems like you left in some testing code? Not a huge deal, but I wonder if you could switch it back to bing for a more general audience?